task9/use_cases/meteorMapper.js

```diff
@@ -0,0 +1,33 @@
+import { getMeteorData } from '../repository/meteorsClient.js'
+
+export const getMeteorDto = async (startDate, endDate, wereDangerousMeteors, count) => {
+    try {
```

Comment:
You can remove try..catch here and leave it only in the controller

Reply:
done